### PR TITLE
Docs: Update downloads

### PR DESCRIPTION
**Pull request by ncbaratta**
### Type of change

- Documentation

### Description

In order to be able to include this section in downstream repositories an attribute needed to be added and the documentation updated with some conditionals.

### CODE REVIEW

File: documentation/common/attributes.adoc [hunk 1]
```diff
@@ -28,6 +28,7 @@
 
 // Source and download links
 :ReleaseDownload: https://github.com/strimzi/strimzi-kafka-operator/releases[GitHub^]
+:ExampleYAMLURL: https://github.com/strimzi/strimzi-kafka-operator/tree/master/examples[Example YAML files^]
```
Comment (scholzj) [inline documentation/common/attributes.adoc]: I think this should not be defined upstream. Or if it is, we need to replace the link with the release version.

**Comment by scholzj** [comment]
@ncbaratta Should this be merged?

**Comment by ncbaratta** [comment]
No, we never finished it. I think I asked @dmuntima to take over ... or I forgot to ask her and I meant to.

I wanted to address your feedback.  Let me see if I can fit it in today.

**Comment by scholzj** [comment]
Ok. I'm not pushing you to do this today ... it can wait for when you or Divya have time. I was just wondering whether it was forgotten or not ;-)

**Comment by ncbaratta** [comment]
No worries @scholzj I appreciate the nudge :)  

If Travis passes then we can merge if you approve of the changes.

**Pull request merged**